tests/unit/conditions/test_return_value.py

```diff
@@ -1,14 +1,23 @@
+import os
 import pytest
 
 from nucypher.policy.conditions.lingo import ReturnValueTest
 
 
 def test_return_result_test_simple():
-    test = ReturnValueTest(comparator='>', value=0)
-    assert test.eval(1)
-    assert not test.eval(-1)
+    test = ReturnValueTest(comparator='>', value='0')
+    assert test.eval('1')
+    assert not test.eval('-1')
```

Comment:
Shall we continue to require string exclusively?  I see this kind of type casting spreading in the codebase a bit.

Reply:
I created a test that shows that literal values are accepted too.